robosuite/environments/two_arm_peg_in_hole.py

```diff
@@ -264,8 +264,9 @@ def _load_model(self):
             self.model.merge(robot.robot_model)
 
         # initialize objects of interest
-        self.mujoco_objects = OrderedDict()
-        self.hole = PlateWithHoleObject()
+        self.hole = PlateWithHoleObject(
+            name="hole",
+        )
         self.cylinder = CylinderObject(
             name="cylinder",
```

Comment:
rename to "peg"?

Reply:
I am a bit confused about the `Task` abstraction. Is `TableTopTask` bound with Table arena? `TwoArmPegInHole` with seems to be an outlier of the TableTopTask. (Also I had no idea from what time the green peg became black after refactoring)

Reply:
So I've had nothing to do with the development of the `TwoArmPegInHole` task so re: the naming, perhaps @cremebrule could handle whatever you had in mind here. As for the `TableTopTask` - it is not bound to any one arena - different arenas can be used - you can see that's the case for `PickPlace`.

Reply:
I haven't touched `TwoArmPegInHole` either, so not sure why the colors have changed. I'll fix it directly on v1.0 branch

Reply:
what exactly did you want to rename to peg? The cylinder object?

Reply:
Yes. The task is called peg in hole and the cylinder object is the peg